### PR TITLE
LGCS-773 - Create appropriate tests for a failed login using invalid …

### PR DESCRIPTION
**Pull request by joel-stobart**
…credentials

### CODE REVIEW

File: acceptance/helpers/FirefoxDriver.js [hunk 1]
```diff
@@ -50,8 +50,16 @@ class FirefoxDriver {
         }
     }
 
+    getUsername() {
+        return this.username;
+    }
+
+    async getDriver() {
+        await this.driver;
+    }
```
Comment (kingdomlevel) [inline acceptance/helpers/FirefoxDriver.js:59]: If this isn't being used can we get rid?
```suggestion
```

**Review by kingdomlevel — APPROVED** [review]
Looks good to me and all passing locally. The login tests are definitely slower now but slow + steady wins the race 🏆🐢🏁........🐇

See suggestion about that getter function - no worries if we want to keep it in for now.

File: acceptance/helpers/FirefoxDriver.js (hunk 1 above)
Comment (joel-stobart) [reply acceptance/helpers/FirefoxDriver.js:59]: I don't have a strong opinion, can we get it merged?

Comment (joel-stobart) [reply acceptance/helpers/FirefoxDriver.js:59]: I'll drop that out and pull in development 👍

**Review by kingdomlevel — APPROVED** [review]

**Pull request merged**